### PR TITLE
Linker tester

### PR DESCRIPTION
**Pull request by NinjaManatee**
Simple linker tester component to include with the seed project. Component can be linked to other components, specify the data type and data to be sent over the linker. 
image

### CODE REVIEW

File: configs/application/components.json [hunk 1]
```diff
@@ -1,171 +1,208 @@
...
+                    "Window Manager": {
+                        "FSBLHeader": false,
+                        "persistWindowState": false
+                    }
+                }
+            }
+        },
+        "Linker Tester": {
+            "window": {
+                "url": "$applicationRoot/components//linkerTester/linkerTester.html",
```
Comment (carterbs) [inline configs/application/components.json]: double slash

File: src-built-in/components/linkerTester/linkerTester.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+			titleDiv.appendChild(document.createTextNode(title));
+
+			const dataBody = document.createElement("pre");
+			dataBody.innerHTML = data;
+
+			received.appendChild(titleDiv);
+			received.appendChild(dataBody);
+		};
+
+		FSBL.addEventListener("onReady", () => {
```
Comment (carterbs) [inline src-built-in/components/linkerTester/linkerTester.js]: For the sake of readability I'd request that you take these callbacks and put them into named functions. We're making a hard push for readability on the other side, so I think we should push for it here as well.

File: src-built-in/components/linkerTester/linkerTester.js [hunk 3]
```diff
@@ -1,94 +1,150 @@
...
+		currentDataType = dataType.value;
+	};
+
+	/**
+	 * Handles the callback for getComponentState
+	 * @param {*} err 
+	 * @param {*} state 
+	 */
+	const componentStateHandler = (err, state) => {
+		return;
```
Comment (carterbs) [inline src-built-in/components/linkerTester/linkerTester.js]: ????

Comment (carterbs) [reply src-built-in/components/linkerTester/linkerTester.js]: why return at the top?

Comment (NinjaManatee) [reply src-built-in/components/linkerTester/linkerTester.js]: Debugging code. Fixed.

**Comment by thorsent** [comment]
The LinkerTester was moved into the finsemble-dev repo

**Pull request closed without merging**